src/rpcserver.cpp

```diff
@@ -26,8 +26,9 @@
 #include "global.h"
 #include "rpcserver.h"
 
-CRpcServer::CRpcServer ( QObject* parent, int iPort, QString strSecret ) :
+CRpcServer::CRpcServer ( QObject* parent, QString strBindIP, int iPort, QString strSecret ) :
     QObject ( parent ),
+    strBindIP ( strBindIP ),
```

Comment:
Can you validate this and issue an error if it's syntactically rubbish, please.

Reply:
The way legacy code has been "validating" IP's (primarily in main.cpp) is by creating an instance of QHostAddress and then using setAddress() with the candidate string.   However, while it catches range errors such as 300.200.100.50, it does NOT catch string-related issues like "192.168. 1.00" (note spaces), and these fall through to the downstream network calls where they eventually throw an error.   I can certainly add parity code in main.cpp to be consistent with other provided addresses, but it doesn't do anything that isn't also caught by the network calls later.

Reply:
OK, I was pretty sure it wasn't too good, if it had any effect at all.  Not to worry then, I think.